ndc-reference/tests/query/predicate_with_exists_in_nested_collection/request.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "../../../../ndc-models/tests/json_schema/query_request.jsonschema",
+    "collection": "institutions",
+    "arguments": {},
+    "query": {
+        "fields": {
+            "id": {
+                "type": "column",
+                "column": "id"
+            },
+            "name": {
+                "type": "column",
+                "column": "name"
+            },
+            "staff": {
+                "type": "column",
+                "column": "staff",
+                "arguments": {
+                    "limit": {
+                        "type": "literal",
+                        "value": null
+                    }
+                }
+            }
+        },
+        "predicate": {
+            "type": "exists",
+            "in_collection": {
+                "type": "nested_collection",
+                "arguments": {
+                    "limit": {
+                        "type": "literal",
+                        "value": null
+                    }
+                },
+                "column_name": "staff"
+            },
+            "predicate": {
```

Comment:
Just checking my understanding - right now the example filters `last_name` containing "s", if we wanted to check for `specialities` that include an "s" instead, we'd nest another `exists` with a `nested_collection` here and put the `binary_comparison_operator` inside that?

Reply:
Correct, but we can only use `exists` on nested _collections_, not nested arrays of strings. You would need to use something like a `contains` operator for your other example.